pages/documentation/migration-V3.md

```diff
@@ -483,68 +480,64 @@ We improved the accessibility of the footer by converting a non-semantic heading
 ### 5. Update to Sass module syntax
 
 {:.border-top-2px.border-base-lighter.padding-top-1}
-USWDS 3.0 supports the deprecated Sass import syntax. You don't need to change to the new syntax in your project Sass to use USWDS 3.0. But once you update, we recommend updating to the new syntax as well.  You’ll need to both switch `@import` to `@use`, and then create `@use` project-specific settings.  In USWDS 3.0 with Sass modules, instead of having multiple theme files that contain every setting available to the design system, you only keep track of the settings you changed in a single file. 
 
-We recommend creating a new file to keep track of these changed settings. These instructions will help you create that file, format it properly, and use it in your project. 
+While USWDS 3.0 supports the deprecated `@import` Sass syntax, we recommend upgrading to the new Sass modules syntax when updating to USWDS 3.0.
 
-You can complete these changes by taking the following steps: 
+Using Sass module syntax allows you to declare theme settings in a clearer, simpler way. Instead of managing multiple lengthy settings files that declare every system setting, you can now use Sass module syntax to declare only the settings you wish to customize, and do it all in a single file.
 
-1. Replace all instances of `@import` with `@forward`, using find/replace in your Sass file.
-2. Locate your existing project theme files. These are the _uswds-theme files that you found in [Step 1: Check your current USWDS code and settings versions](#1-check-your-current-uswds-code-and-settings-versions). (If you already have your project-specific settings file in a single file, you can skip ahead Step 4. If not, you'll need to collect your project-specific settings in a single file.)
-3. In the project theme file directory directory, create a new file called `_uswds-theme.scss`.
+These instructions will help you first update your `@import` references, then will show you how to create a new `_uswds-theme` file, format it properly, and use it in your project.  
 
-### 6. Find which settings from your theme files you've customized.
+1. **Replace all instances of @import with @forward in your Sass entry point.** Update all of the `@import` references in your Sass entry point to `@forward`. 
```

Comment:
I limited the scope of this instruction to just reference the Sass entry point. Suggesting a global replacement felt risky, since there would likely be appropriate times to use `@use` instead. If that is inaccurate, kindly disregard.